src/components/NewSpaceCwForm.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import styles from '../assets/stylesheets/newSpaceCwForm.module.css';
+
+const NewSpaceCwForm = () => {
+  const [formData, setFormData] = useState({
+    name: '',
+    model: '',
+    description: '',
+    address: '',
+    price: '',
+    image: '',
+    discount: '',
+    category: '',
+    user_id: '1',
+  });
+
+  const [successMessage, setSuccessMessage] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await axios.post('http://localhost:3001/api/v1/users/:user_id/space_cws', formData);
+      setSuccessMessage('New coworking space created successfully!');
+      setTimeout(() => {
+        setSuccessMessage('');
+      }, 5000); // 5 seconds
+    } catch (error) {
+      setErrorMessage('Error creating coworking space. Please try again later.');
+      setTimeout(() => {
+        setErrorMessage('');
+      }, 8000); // 8 seconds
+    }
+  };
+
+  return (
+    <div className={styles.newCwsContainer}>
+      <div className={styles.newCwsOverlay}>
+        <h2 className={styles.newCwsTitle}>Create a New Coworking Space</h2>
+        <form onSubmit={handleSubmit} className={styles.formContainer}>
+          {/* <label htmlFor="name">Name:</label> */}
+          <input
+            type="text"
+            name="name"
+            className="text"
+            placeholder="Name:"
+            value={formData.name}
+            onChange={handleChange}
+            required
+          />
+
+          {/* <label htmlFor="model">Model:</label> */}
```

Comment:
Hi @kazim110 . Could you please delete the 8 <label></label> tags that are not used?